### PR TITLE
R&D Tweaks V.2 - Dev Branch Edition

### PR DESCRIPTION
**Pull request by AllCorruptingWyrm**
## About The Pull Request
The following PR removes printable weapons like Gauss Rifles and Plasma Rifles from R&D, in turn they get weapon parts which might need to be evaluated overtime but we'll see how it goes. Following that, this adds plasma cutters to R&D and removes dynamite but lowers costs on caps so town miners aren't out of a job and aren't blowing up the entire map or something similar.

## Pre-Merge Checklist
- [X] You tested this on a local server.
- [X] This code did not runtime during testing.
- [X] You documented all of your changes.

## Changelog
:cl:
add: weapon parts
add: RCD to R&D & Autolathes
remove: weapons to R&D
/:cl:

### CODE REVIEW

**Comment by Zero-Percent-Angel** [comment]
Hey could you merge master back into this then change the base branch to be dev! That way I can see the changes a bit easier :)

File: code/modules/research/designs/weapon_designs.dm [hunk 1]
```diff
@@ -306,7 +306,7 @@
 	build_path = /obj/item/gun/ballistic/revolver/thatgun
 	category = list("Weapons")
 	departmental_flags = DEPARTMENTAL_FLAG_SECURITY
-
+/*
 /datum/design/n99
```
Comment (Zero-Percent-Angel) [inline code/modules/research/designs/weapon_designs.dm:310]: Did you leave the blueprint references hanging for this? I can't see any changes in that area...

With that said these designs here come from the science lab I am going to make some adjustments in that area to do with how much it costs to create blueprints; as well as revising the research points gained on experiment.

**Comment by Zero-Percent-Angel** [comment]
Do you mind if we park some of the design removals in this PR while I work on science lab adjustments?

**Comment by AllCorruptingWyrm** [comment]
> Do you mind if we park some of the design removals in this PR while I work on science lab adjustments?

oh sure go ahead, this PR was more of an emergency thing because I saw 20 plas rifles+amrs lol, make whatever suggestions you'll want and I'll resolve them.

**Comment by VEN0M5** [comment]
They can still print out the parts and make em manually through the crafting menu after reading the BP from the untouched sci lab.

**Comment by AllCorruptingWyrm** [comment]
> They can still print out the parts and make em manually through the crafting menu after reading the BP from the untouched sci lab.

angel said s/he(?) was adjusting the lab

**Comment by Zero-Percent-Angel** [comment]
I did change the lab to make weapons 10 times more expensive in terms of points when trying to create blueprints

**Pull request closed without merging**